addons/mail/static/src/js/form_renderer.js

```diff
@@ -2,20 +2,31 @@ odoo.define('mail.form_renderer', function (require) {
 "use strict";
 
 var Chatter = require('mail.Chatter');
+var OwlChatter = require('mail.component.Chatter');
 var FormRenderer = require('web.FormRenderer');
 
 /**
  * Include the FormRenderer to instanciate the chatter area containing (a
  * subset of) the mail widgets (mail_thread, mail_followers and mail_activity).
  */
 FormRenderer.include({
+    dependencies: ['owl'],
+    on_attach_callback: function () {
+        this.chatter_component.__callMounted();
+    },
+    on_detach_callback: function () {
+        this.chatter_component.__callWillUnmount();
+    },
+
```

Comment:
should not be necessary if chatter component is mounted correctly. Note that destroying form renderer should also destroy chatter component!

Reply:
I think we should talk about how to do that properly because I'm sure it's not the right way but it's the only way I found to make it work. As it worked, I left it like that to allow me to theme the chatter.

Reply:
I made all other changes and still without these lines it does not work..